ffmpeg/ffmpeg.go

```diff
@@ -27,6 +33,37 @@ func RTMPToHLS(localRTMPUrl string, outM3U8 string, tmpl string, seglen_secs str
 	return nil
 }
 
+func Transcode(input string, ps []VideoProfile) error {
+	inp := C.CString(input)
+	params := make([]C.output_params, len(ps))
+	for i, param := range ps {
```

Comment:
I think we have to put the transcode call in a go routine somewhere in this call stack.  Otherwise it'll block until the work finishes.  Maybe the best place is in `Livepeernode.TranscodeAndBroadcast()` - just pointing it out so we don't forget.

Reply:
Sure -- I believe the old behavior was synchronous as well. If the blocking has been a problem with with go-livepeer all along, then we can change it there.